dashboard/test/helpers/initial_section_creation_interstitial_helper_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+# Tests following spec:
+# We should pop up this interstitial only for teachers on their first sign-in.
+class InitialSectionCreationInterstitialHelperTest < ActiveSupport::TestCase
```

Comment:
Yay for thoughtful test cases! Do I remember correctly that there's some interplay between this and the other interstitial dialog? And if so, is there any world in which we would want an eyes test to make sure the right one appears (slash they appear in the right order)?

Reply:
I couldn't find test UI coverage for the Clever integration, so I'm hoping to just do manual tests here. Looks like there is a bug with Clever sign-in right now, so I checked in with Platform and will hopefully be able to do that soon.

Reply:
Alrighty! Manually tested with Clever integration. We do prioritize the Clever-specific dialog, as expected. Let me know if you think more testing is necessary or if we're good to go!

Reply:
I think ideally, we'd have a .feature file for this. But since we never made a test file in the first place, I'm cool with it being a follow up task (as it would be a bigger undertaking)!

Reply:
I'll track one in Jira!